scripts/create-discussions.py

```
@@ -0,0 +1,96 @@
+    "mona-lisa": "Mona Lisa",
+    "aurora-sunset": "Aurora Sunset",
+}
+
+headers = {
+    "Authorization": f"Bearer {TOKEN}",
+    "Content-Type": "application/json",
+    "Accept": "application/vnd.github+json"
+}
+
+# Check existing discussions
+resp = requests.post(API_URL, headers=headers, json={
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)